batchflow/models/tf/base.py

```diff
@@ -1587,13 +1587,13 @@ def head(cls, inputs, name='head', **kwargs):
             return conv_block(inputs, name=name, **kwargs)
         return inputs
 
-    def output(self, inputs, predictions=None, ops=None, prefix=None, **kwargs):
+    def output(self, inputs, predictions=None, ops=None, **kwargs):
```

Comment:
Can you elaborate on what is going on here besides some code refactoring (better name making for added output `ops` and removing redundant `prefix` argument)?

Reply:
Previously, output-block processed multiple outputs from `head` in wrong way.
Now all outputs from `head` will be transformed by `ops` dict and in case of multiple outputs each output will have postfix `_{i}` where i is an index.